aspnetcore/fundamentals/servers/weblistener/sample/project.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "Microsoft.AspNetCore.Server.WebListener": "1.0.0",
```

Comment:
I would use the latest LTS minor versions even if we decide not to update the docs to use 1.1.